File: tema_pclp_c_lab4/ex2.c
#include <stdio.h>
int main()
{
    int N;
    printf("Va rog introduceti un numar: ");
    scanf("%d", &N);
    int v[N];
    for (int i = 0; i < N; i++)
        scanf("%d", &v[i]);
    for (int i = 0; i < N - 1; i++)
        for (int j = i + 1; j < N; j++)
            if (v[i] > v[j]) {
                v[i] = v[i] ^ v[j];
                v[j] = v[i] ^ v[j];
                v[i] = v[j] ^ v[i];
            }
    for (int i = 0; i < N; i++)
        printf("%d ", v[i]);
}

File: tema 2_java/src/ex2.java
import java.io.*;
public class ex2 {
    public static void main(String[] args) throws IOException{
        System.out.println("Va rog introduceti un numar: ");
        InputStreamReader isr = new InputStreamReader(System.in);
        BufferedReader br = new BufferedReader(isr);
        String s = br.readLine();
        int n = Integer.parseInt(s);
        int maxim = -1;
        int minim = 11;
        while (n != 0){
            if (n % 10 > maxim)
                maxim = n % 10;
            if (n % 10 < minim)
                minim = n % 10;
            n = n / 10;
        }
        System.out.print("Cea mai mare cifra este " + maxim + " si cea mai mica este " + minim);
    }
}
